src/SignIn/index.js

```diff
@@ -33,13 +31,16 @@ function SignIn(props) {
             onChange={(e) =>
               setCredentials({ ...credentials, username: e.target.value })
             }
-          /> <br />
+          />{" "}
```

Comment:
is the `{" "}` intentional ik it got automatically added to my stuff a lot, if it is that's cool